libs/librepcb/common/graphics/graphicsview.cpp

```diff
@@ -139,12 +139,12 @@ Point GraphicsView::mapGlobalPosToScenePos(const QPoint& globalPosPx,
 }
 
 void GraphicsView::handleMouseWheelEvent(
-    QGraphicsSceneWheelEvent* event) noexcept {
-  if (event->modifiers().testFlag(Qt::ShiftModifier)) {
+    QWheelEvent* event) noexcept {
+  if (event->buttons() & Qt::ShiftModifier) {
```

Comment:
This is not allowed since `event->buttons()` has the return type `Qt::MouseButtons`, while `Qt::ShiftModifier` is of type `Qt::KeyboardModifier`. Same issue a few lines below. These changes break horizontal and vertical scrolling.

And why was the type of the parameter `event` changed from `QGraphicsSceneWheelEvent*` to `QWheelEvent?`?

Reply:
I've removed this part of the diff. It's now back to the original design.